### PR TITLE
add or filter along with tests

### PR DESCRIPTION
**Pull request by silentworks**
## What kind of change does this PR introduce?

Add `or` filter along with unit and integration tests

## What is the current behavior?

No `or` filter

## What is the new behavior?

Has `or` filter

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: infra/docker-compose.yaml [hunk 1]
```diff
@@ -20,6 +20,6 @@ services:
       POSTGRES_USER: app_user
       POSTGRES_PASSWORD: password
 # Uncomment this if you want to persist the data. Create your boostrap SQL file in the project root
-#    volumes:
-#      - "./pgdata:/var/lib/postgresql/data"
-#      - "./init.sql:/docker-entrypoint-initdb.d/init.sql"
+    volumes:
+      # - "./pgdata:/var/lib/postgresql/data"
```
Comment (J0) [inline infra/docker-compose.yaml:24]: Guessing we're keeping this for local testing?

**Review by J0 — DISMISSED** [review]
Nice,

Thanks for the tests!

File: infra/docker-compose.yaml (hunk 1 above)
Comment (silentworks) [reply infra/docker-compose.yaml:24]: Yeah thats the only use for that, we could probably remove it later as I haven't used it myself.

File: postgrest/base_request_builder.py [hunk 2]
```diff
@@ -330,6 +330,17 @@ def ilike(self: Self, column: str, pattern: str) -> Self:
         """
         return self.filter(column, Filters.ILIKE, pattern)
 
+    def or_(self: Self, filters: str, reference_table: Union[str, None] = None) -> Self:
```
[the remark this answers is not in the document]
Comment (silentworks) [reply postgrest/base_request_builder.py:333]: Yes that's correct.

**Pull request merged**